types/node-resque/node-resque-tests.ts

```diff
@@ -64,9 +64,41 @@ worker.connect().then(() => worker.start());
 // start scheduler
 scheduler.connect().then(() => scheduler.start());
 
-queue.connect().then(() => {
+queue.connect().then(async () => {
```

Comment:
Can we also add the following snippet after line 24, similar to #40027 to make it fully consistent to @gfoltz changes.

```ts
 const connectionWithCustomRedis: ConnectionOptions = { redis: "fake_redis_client" };
```

Reply:
Updated in https://github.com/DefinitelyTyped/DefinitelyTyped/pull/39973/commits/45a3df7a5e9f1801a0722d4857f842d4dced6b99

Reply:
👍 Hopefully failing CI would be figured out soon and you can merge then. 🙏